app/controllers/agenda/minutes.js

```diff
@@ -7,11 +7,13 @@ import constants from 'frontend-kaleidos/config/constants';
 import { SECRETARIS_NAME } from 'frontend-kaleidos/config/config';
 import { task as trackedTask } from 'ember-resources/util/ember-concurrency';
 import { dateFormat } from 'frontend-kaleidos/utils/date-format';
+import { deleteFile } from 'frontend-kaleidos/utils/document-delete-helpers';
+import VRDocumentName from 'frontend-kaleidos/utils/vr-document-name';
 
 function renderAttendees(attendees) {
```

Comment:
Lots of these functions outside the controller would be better as utils and/or getters inside the controller. But we'll handle this later so this can be tested.

Reply:
Why?